### PR TITLE
Feature/no false classes

### PR DESCRIPTION
**Pull request by rin**
Before:
Screenshot 2020-04-03 at 16 54 58
After:
Screenshot 2020-04-03 at 16 48 47

### CODE REVIEW

**Comment by camposcristianeze** [comment]
This looks like a great refactor :) Just for some context, is this because there were classes that we stopped to use/delete before and they're returning either `null` , `false` or `undefined` ? 

Then, if that's the case, @kbardi maybe we can think of adding a test that runs exactly those 3 lines that Rin shared ?

**Comment by rin** [comment]
@kbardi Can I have a review here? And … I'm not sure how to add what Cris asked, but it would be a nice idea …?

File: src/lib/components/input/input.js [hunk 1]
```diff
@@ -36,10 +37,10 @@ const Input = ({
     handleInputChange(event, name, handleValidations, onChangeFunction);
 
   return (
-    <div className={`${styles[`theme-${theme}`]} ${containerClassName}`}>
+    <div className={classNames(styles[`theme-${theme}`], containerClassName)}>
       <div className={styles.wrapper}>
         <input
-          className={`${className} ${isValidationOk() && styles.error}`}
+          className={`${className} ${isValidationOk() ? styles.error : ''}`}
```
Comment (kbardi) [inline src/lib/components/input/input.js:43]: if you use classNames, you could do: `className={classNames(className, { [styles.error]: isValidationOk() })}`

File: src/lib/components/select/select.js [hunk 2]
```diff
@@ -99,12 +103,15 @@ class Select extends PureComponent {
...
+        className={classNames(
+          containerClassName,
+          stylesCss[`theme-${theme}`],
+          stylesCss.wrapper,
+        )}
       >
         <Dropdown
           placeholder={placeholder}
-          className={`${className} ${isValidationOk() && styles.error}`}
+          className={`${className} ${isValidationOk() ? styles.error : ''}`}
```
Comment (kbardi) [inline src/lib/components/select/select.js:114]: Same here, if you are using classNames, it is better to add the condition inside classNames, like:
`className={classNames(className, { [styles.error]: isValidationOk() })}`

**Comment by rin** [comment]
@camposcristianeze Had a quick look, and not sure how we should do the check (would probably good to check it for each rendered component individually?), so I opened a new issue for it: https://github.com/aimementoring/blueprint/issues/201

**Pull request merged**